fdbserver/VersionedBTree.actor.cpp

```diff
@@ -2629,6 +2815,16 @@ class DWALPager : public IPager2 {
 			debug_printf("DWALPager(%s) Set the cacheEntry readFuture for page: %s\n",
 			             filename.c_str(),
 			             toString(pageID).c_str());
+
+			++g_redwoodMetrics.metric.pagerCacheMiss;
+			auto metrics = g_redwoodMetrics.level(l).metric;
```

Comment:
This and similar statements should be `auto &` or I'm pretty sure it's just making a local copy and then modifying it.  Also, since both statements using `metrics` are using `eventReasons` so use 
`auto &eventReasons = g_redwoodMetrics.level(l).metric.eventReasons`